2017-08-06 19:26:55,620 [Thread-34:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.312) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 19:26:56,994 [Thread-34:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.422) SELECT * FROM cryapp_cryorder WHERE Status = 1 GROUP BY GoodId_id; args=()
2017-08-06 19:26:57,372 [Thread-34:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.361) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:26:57,416 [Thread-34:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 7908
2017-08-06 19:27:20,628 [Thread-35:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:27:20,645 [Thread-35:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:27:20,750 [Thread-35:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.024) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:27:20,781 [Thread-35:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:27:20,836 [Thread-35:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:27:20,912 [Thread-38:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/js/js_sdk20170302.js HTTP/1.1" 200 10139
2017-08-06 19:27:20,919 [Thread-37:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/application-5983038036a092579f0a61ac4dcfa044.css HTTP/1.1" 200 321854
2017-08-06 19:27:21,668 [Thread-39:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/img/mk_pro_cover.jpg HTTP/1.1" 200 70172
2017-08-06 19:27:21,678 [Thread-40:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/fonts/ios7-style-font-icons-22de609b714050cb49679bc4054365e6.ttf HTTP/1.1" 200 197548
2017-08-06 19:27:23,581 [Thread-41:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/favicon-07e579901353a57d289b9b754f0e2912.ico HTTP/1.1" 200 1150
2017-08-06 19:27:25,147 [Thread-42:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/css/material-kit.css.map HTTP/1.1" 200 45976
2017-08-06 19:35:01,777 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:35:01,805 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT VERSION(); args=None
2017-08-06 19:35:02,002 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.087) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:35:02,086 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.033) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:35:02'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:35:02')
2017-08-06 19:35:02,236 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.099) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:35:02,393 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.080) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:35:02,549 [Thread-43:123145372995584] [django.db.backends:90] [utils:execute] [DEBUG]- (0.067) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:35:02' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:35:02', 0)
2017-08-06 19:35:23,486 [Thread-43:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "POST /goods/1501998367719358 HTTP/1.1" 302 0
2017-08-06 19:35:23,709 [Thread-44:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /webbrowser/ HTTP/1.1" 200 11067
2017-08-06 19:35:23,884 [Thread-45:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/css/bootstrap.min.css HTTP/1.1" 200 109998
2017-08-06 19:35:23,979 [Thread-46:123145383505920] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/css/material-kit.css HTTP/1.1" 200 113645
2017-08-06 19:35:24,049 [Thread-47:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/jquery.min.js HTTP/1.1" 200 93868
2017-08-06 19:35:24,130 [Thread-49:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/bootstrap.min.js HTTP/1.1" 200 36816
2017-08-06 19:35:24,186 [Thread-50:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/material.min.js HTTP/1.1" 200 8137
2017-08-06 19:35:24,237 [Thread-51:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/nouislider.min.js HTTP/1.1" 200 12944
2017-08-06 19:35:24,272 [Thread-52:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/bootstrap-datepicker.js HTTP/1.1" 200 14137
2017-08-06 19:35:24,344 [Thread-53:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/js/material-kit.js HTTP/1.1" 200 3693
2017-08-06 19:35:24,412 [Thread-54:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/zhesstwocode.png HTTP/1.1" 200 1117
2017-08-06 19:35:24,475 [Thread-55:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/img/avatar.jpg HTTP/1.1" 200 85214
2017-08-06 19:35:24,530 [Thread-56:123145378250752] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/img/christian.jpg HTTP/1.1" 200 80546
2017-08-06 19:35:24,577 [Thread-57:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/img/kendall.jpg HTTP/1.1" 200 81432
2017-08-06 19:35:25,862 [Thread-58:123145372995584] [django.server:131] [basehttp:log_message] [INFO]- "GET /static/assets/css/material-kit.css.map HTTP/1.1" 200 45976
2017-08-06 19:35:49,316 [Dummy-6:123145368145920] [django.db.backends:90] [utils:execute] [DEBUG]- (0.300) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:35:49,605 [Dummy-6:123145368145920] [django.db.backends:90] [utils:execute] [DEBUG]- (0.287) SELECT VERSION(); args=None
2017-08-06 19:35:51,641 [Dummy-6:123145368145920] [django.db.backends:90] [utils:execute] [DEBUG]- (0.130) SHOW FULL TABLES; args=None
2017-08-06 19:35:51,826 [Dummy-6:123145368145920] [django.db.backends:90] [utils:execute] [DEBUG]- (0.166) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:36:00,596 [Thread-7:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.274) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:36:00,866 [Thread-7:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.268) SELECT VERSION(); args=None
2017-08-06 19:36:02,826 [Thread-7:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.420) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:36:03,173 [Thread-7:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.337) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:36:03,287 [Thread-7:123145373401088] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:36:09,252 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:36:09,274 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT VERSION(); args=None
2017-08-06 19:36:09,397 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.031) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:36:09,430 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:36:09'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:36:09')
2017-08-06 19:36:09,502 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.034) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:36:09,553 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.035) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:36:09,595 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.026) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:36:09' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:36:09', 0)
2017-08-06 19:36:22,590 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:36:25,282 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:36:25,301 [Thread-14:123145373401088] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:36:58,551 [Dummy-6:123145344131072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:36:58,574 [Dummy-6:123145344131072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:37:02,523 [Dummy-6:123145344131072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.064) SHOW FULL TABLES; args=None
2017-08-06 19:37:02,568 [Dummy-6:123145344131072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:37:08,917 [Dummy-6:123145472192512] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:37:08,935 [Dummy-6:123145472192512] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:37:11,783 [Dummy-6:123145472192512] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SHOW FULL TABLES; args=None
2017-08-06 19:37:11,815 [Dummy-6:123145472192512] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:37:20,116 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:37:20,134 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 19:37:20,249 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:37:20,285 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:37:20'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:37:20')
2017-08-06 19:37:20,384 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.068) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:37:20,448 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.037) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:37:20,542 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:37:20' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:37:20', 0)
2017-08-06 19:37:42,031 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:37:43,044 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:37:43,062 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:37:51,097 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.396) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:37:51,587 [Thread-7:123145477447680] [django.db.backends:90] [utils:execute] [DEBUG]- (0.485) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 86; args=(86,)
2017-08-06 19:38:53,831 [Dummy-6:123145444122624] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:38:53,848 [Dummy-6:123145444122624] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:38:55,053 [Dummy-6:123145444122624] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 19:38:55,084 [Dummy-6:123145444122624] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:39:00,047 [Thread-7:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:39:00,065 [Thread-7:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 19:39:00,169 [Thread-7:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.028) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:39:00,203 [Thread-7:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:39:00,369 [Thread-7:123145449377792] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:39:09,168 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:39:09,192 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 19:39:09,340 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.032) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:39:09,390 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:39:09'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:39:09')
2017-08-06 19:39:09,502 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.042) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:39:09,544 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.024) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:39:09,591 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.030) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:39:09' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:39:09', 0)
2017-08-06 19:39:16,273 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:39:16,881 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:39:16,900 [Thread-9:123145449377792] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:40:01,001 [Dummy-6:123145409904640] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:40:01,017 [Dummy-6:123145409904640] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:40:02,240 [Dummy-6:123145409904640] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 19:40:02,270 [Dummy-6:123145409904640] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:40:06,529 [Thread-7:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:40:06,550 [Thread-7:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 19:40:06,693 [Thread-7:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.047) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:40:06,798 [Thread-7:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.070) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:40:07,166 [Thread-7:123145415159808] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:40:14,384 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:40:14,403 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 19:40:14,511 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:40:14,543 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:40:14'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:40:14')
2017-08-06 19:40:14,616 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.046) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:40:14,696 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.041) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:40:14,761 [Thread-9:123145415159808] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:40:14' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:40:14', 0)
2017-08-06 19:40:49,494 [Dummy-6:123145550331904] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:40:49,512 [Dummy-6:123145550331904] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 19:40:51,008 [Dummy-6:123145550331904] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 19:40:51,038 [Dummy-6:123145550331904] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:40:54,930 [Thread-7:123145555587072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:40:54,951 [Thread-7:123145555587072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT VERSION(); args=None
2017-08-06 19:40:55,079 [Thread-7:123145555587072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.051) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:40:55,201 [Thread-7:123145555587072] [django.db.backends:90] [utils:execute] [DEBUG]- (0.090) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:40:55,567 [Thread-7:123145555587072] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:41:02,861 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:41:02,907 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.043) SELECT VERSION(); args=None
2017-08-06 19:41:03,021 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:41:03,067 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:41:03'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:41:03')
2017-08-06 19:41:03,235 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.136) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:41:03,313 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.062) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:41:03,519 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.161) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:41:03' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:41:03', 0)
2017-08-06 19:41:06,404 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.252) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:41:07,234 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:41:07,254 [Thread-8:123145560842240] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:41:43,854 [Dummy-6:123145507708928] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:41:43,871 [Dummy-6:123145507708928] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:41:45,142 [Dummy-6:123145507708928] [django.db.backends:90] [utils:execute] [DEBUG]- (0.043) SHOW FULL TABLES; args=None
2017-08-06 19:41:45,227 [Dummy-6:123145507708928] [django.db.backends:90] [utils:execute] [DEBUG]- (0.076) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:41:56,151 [Thread-7:123145512964096] [django.db.backends:90] [utils:execute] [DEBUG]- (0.034) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:41:56,170 [Thread-7:123145512964096] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 19:41:56,281 [Thread-7:123145512964096] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:41:56,315 [Thread-7:123145512964096] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:41:56,430 [Thread-7:123145512964096] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:42:01,178 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:42:01,196 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 19:42:01,311 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.034) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:42:01,357 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.032) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:42:01'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:42:01')
2017-08-06 19:42:01,444 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.060) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:42:01,513 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.042) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:42:01,588 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.024) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:42:01' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:42:01', 0)
2017-08-06 19:42:07,461 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:42:07,483 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:42:07,506 [Thread-8:123145518219264] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:42:23,520 [Dummy-6:123145326706688] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:42:23,537 [Dummy-6:123145326706688] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 19:42:24,860 [Dummy-6:123145326706688] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 19:42:24,899 [Dummy-6:123145326706688] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:42:28,155 [Thread-7:123145331961856] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:42:28,173 [Thread-7:123145331961856] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:42:28,293 [Thread-7:123145331961856] [django.db.backends:90] [utils:execute] [DEBUG]- (0.032) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:42:28,343 [Thread-7:123145331961856] [django.db.backends:90] [utils:execute] [DEBUG]- (0.030) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:42:28,663 [Thread-7:123145331961856] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:42:35,917 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:42:35,967 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT VERSION(); args=None
2017-08-06 19:42:36,096 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.038) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:42:36,210 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:42:36'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:42:36')
2017-08-06 19:42:36,360 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.073) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:42:36,478 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.069) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:42:36,628 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.040) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:42:36' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:42:36', 0)
2017-08-06 19:42:36,684 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.043) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:42:36,783 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.042) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:42:36,814 [Thread-8:123145337217024] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 19:48:18,228 [Dummy-6:123145424793600] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:48:18,248 [Dummy-6:123145424793600] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT VERSION(); args=None
2017-08-06 19:48:20,660 [Dummy-6:123145424793600] [django.db.backends:90] [utils:execute] [DEBUG]- (0.066) SHOW FULL TABLES; args=None
2017-08-06 19:48:20,708 [Dummy-6:123145424793600] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 19:48:24,922 [Thread-7:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:48:24,943 [Thread-7:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT VERSION(); args=None
2017-08-06 19:48:25,061 [Thread-7:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.039) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 19:48:25,191 [Thread-7:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.112) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 19:48:25,475 [Thread-7:123145430048768] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 19:48:41,193 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.171) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 19:48:41,438 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.244) SELECT VERSION(); args=None
2017-08-06 19:48:42,178 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:48:42,204 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 19:48:42'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 19:48:42')
2017-08-06 19:48:42,246 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.027) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 19:48:42,280 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE (`auth_user`.`id` = 1 AND `auth_user`.`is_blacklist` = 1); args=(1, 1)
2017-08-06 19:48:42,316 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.024) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 19:48:42' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 19:48:42', 0)
2017-08-06 19:48:42,339 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT `shops`.`id`, `shops`.`user_id`, `shops`.`shopname`, `shops`.`shopkeepername`, `shops`.`platform`, `shops`.`add_time` FROM `shops` WHERE `shops`.`id` = '15019983676355634'; args=('15019983676355634',)
2017-08-06 19:48:42,369 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`id` = '150199836775589'; args=('150199836775589',)
2017-08-06 19:48:42,445 [Thread-9:123145430048768] [django.db.backends:90] [utils:execute] [DEBUG]- (0.069) SELECT `users_tbusername`.`id`, `users_tbusername`.`user_id`, `users_tbusername`.`tbUsername`, `users_tbusername`.`is_blacklist` FROM `users_tbusername` WHERE `users_tbusername`.`user_id` = 1; args=(1,)
2017-08-06 20:39:22,870 [Dummy-6:123145430093824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 20:39:22,890 [Dummy-6:123145430093824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT VERSION(); args=None
2017-08-06 20:39:23,751 [Dummy-6:123145430093824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SHOW FULL TABLES; args=None
2017-08-06 20:39:23,777 [Dummy-6:123145430093824] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 20:41:22,343 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>"
2017-08-06 20:41:22,425 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>"
2017-08-06 20:41:22,437 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in '<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>'
2017-08-06 20:41:22,449 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>"
2017-08-06 20:41:22,464 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>"
2017-08-06 20:41:22,478 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>"
2017-08-06 20:41:22,498 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>"
2017-08-06 20:41:22,519 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>"
2017-08-06 20:41:22,526 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 20:41:22,539 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>"
2017-08-06 20:41:22,555 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>"
2017-08-06 20:41:22,558 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 20:41:22,623 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>"
2017-08-06 20:41:22,697 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>"
2017-08-06 20:41:22,748 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>"
2017-08-06 20:41:22,753 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.027) SELECT `django_session`.`session_key`, `django_session`.`session_data`, `django_session`.`expire_date` FROM `django_session` WHERE (`django_session`.`session_key` = '1dm94bmoyvc9jv8kd85bogi44ma6mis8' AND `django_session`.`expire_date` > '2017-08-06 20:41:22'); args=('1dm94bmoyvc9jv8kd85bogi44ma6mis8', '2017-08-06 20:41:22')
2017-08-06 20:41:22,811 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>"
2017-08-06 20:41:22,828 [Thread-10:123145451114496] [django.template:929] [base:_resolve_lookup] [DEBUG]- Exception while resolving variable 'name' in template 'unknown'.
Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/exception.py", line 42, in inner
    response = get_response(request)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/core/handlers/base.py", line 172, in _get_response
    resolver_match = resolver.resolve(request.path_info)
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/urls/resolvers.py", line 300, in resolve
    raise Resolver404({'tried': tried, 'path': new_path})
django.urls.exceptions.Resolver404: {'tried': [[<RegexURLResolver <module 'auto.urls' from '/Users/VANXV/PycharmProjects/zss/auto/urls.py'> (None:auto) ^auto/>], [<RegexURLResolver <module 'grappelli.urls' from '/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/grappelli/urls.py'> (None:None) ^grappelli/>], [<RegexURLResolver <RegexURLPattern list> (admin:admin) ^admin/>], [<RegexURLResolver <module 'blacklist.urls' from '/Users/VANXV/PycharmProjects/zss/blacklist/urls.py'> (None:blacklist) ^blacklist/>], [<RegexURLResolver <module 'goods.urls' from '/Users/VANXV/PycharmProjects/zss/goods/urls.py'> (None:good) ^good/>], [<RegexURLResolver <module 'orders.urls' from '/Users/VANXV/PycharmProjects/zss/orders/urls.py'> (None:order) ^order/>], [<RegexURLResolver <module 'crm.urls' from '/Users/VANXV/PycharmProjects/zss/crm/urls.py'> (None:crm) ^crm/>], [<RegexURLResolver <module 'finance.urls' from '/Users/VANXV/PycharmProjects/zss/finance/urls.py'> (None:finance) ^finance/>], [<RegexURLResolver <module 'cashback.urls' from '/Users/VANXV/PycharmProjects/zss/cashback/urls.py'> (None:cashback) ^cashback/>], [<RegexURLResolver <module 'cryapp.urls' from '/Users/VANXV/PycharmProjects/zss/cryapp/urls.py'> (None:cryapp) ^cryapp/>], [<RegexURLResolver <module 'wechat.urls' from '/Users/VANXV/PycharmProjects/zss/wechat/urls.py'> (None:wechat) m/>], [<RegexURLPattern login ^login/>], [<RegexURLPattern register ^register/>], [<RegexURLPattern logout ^logout/>], [<RegexURLPattern welcome ^welcome/$>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'users.urls' from '/Users/VANXV/PycharmProjects/zss/users/urls.py'> (None:None) ^users/>], [<RegexURLPattern GetGoods ^goods/(?P<goodid>(\d+))>], [<RegexURLPattern webbrowser webbrowser/>], [<RegexURLResolver <module 'financial.urls' from '/Users/VANXV/PycharmProjects/zss/financial/urls.py'> (None:financialurl) ^financial/>], [<RegexURLPattern buyerindex ^$>], [<RegexURLResolver <module 'servers.urls' from '/Users/VANXV/PycharmProjects/zss/servers/urls.py'> (None:None) ^servers/>], [<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>], [<RegexURLPattern None ^media\/(?P<path>.*)$>]], 'path': 'robots.txt'}

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 885, in _resolve_lookup
    current = current[bit]
TypeError: 'RegexURLResolver' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 893, in _resolve_lookup
    current = getattr(current, bit)
AttributeError: 'RegexURLResolver' object has no attribute 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 900, in _resolve_lookup
    current = current[int(bit)]
ValueError: invalid literal for int() with base 10: 'name'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/VANXV/PycharmProjects/venvDjango1.11py3.6/lib/python3.6/site-packages/django/template/base.py", line 907, in _resolve_lookup
    (bit, current))  # missing attribute
django.template.base.VariableDoesNotExist: Failed lookup for key [name] in "<RegexURLResolver <module 'adapi.urls' from '/Users/VANXV/PycharmProjects/zss/adapi/urls.py'> (None:None) ^adapi/>"
2017-08-06 20:41:22,879 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.059) SELECT `auth_user`.`id`, `auth_user`.`password`, `auth_user`.`last_login`, `auth_user`.`is_superuser`, `auth_user`.`username`, `auth_user`.`first_name`, `auth_user`.`last_name`, `auth_user`.`email`, `auth_user`.`is_staff`, `auth_user`.`is_active`, `auth_user`.`date_joined`, `auth_user`.`is_seller`, `auth_user`.`is_blacklist`, `auth_user`.`address`, `auth_user`.`mobile`, `auth_user`.`balance`, `auth_user`.`wechatName`, `auth_user`.`wechat`, `auth_user`.`qq`, `auth_user`.`alipay`, `auth_user`.`bankName`, `auth_user`.`bankID`, `auth_user`.`realname`, `auth_user`.`tags`, `auth_user`.`image`, `auth_user`.`referrercode`, `auth_user`.`remark` FROM `auth_user` WHERE `auth_user`.`id` = 1; args=(1,)
2017-08-06 20:41:22,899 [Thread-10:123145451114496] [django.server:131] [basehttp:log_message] [WARNING]- "GET /robots.txt HTTP/1.1" 404 4585
2017-08-06 20:41:22,928 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.030) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE (`cryapp_cryorder`.`buyerid_id` = 1 AND `cryapp_cryorder`.`AddTime` > '2017-07-04 20:41:22' AND NOT (`cryapp_cryorder`.`Status` = 0)); args=(1, '2017-07-04 20:41:22', 0)
2017-08-06 20:41:23,406 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.024) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:23,430 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:23,467 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:23,491 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:25,521 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.037) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:25,546 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:25,585 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:25,607 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:26,212 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.029) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:26,239 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:26,274 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:26,299 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.023) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,278 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.027) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,319 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,353 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,377 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,922 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,944 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:27,982 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:28,005 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` LIMIT 21; args=()
2017-08-06 20:41:28,958 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.022) SELECT * FROM cryapp_cryorder WHERE Status = 1 GROUP BY GoodId_id; args=()
2017-08-06 20:41:28,997 [Thread-7:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.025) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 20:41:29,139 [Thread-7:123145435348992] [django.server:131] [basehttp:log_message] [INFO]- "GET / HTTP/1.1" 200 7908
2017-08-06 20:41:32,492 [Thread-12:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 20:41:32,509 [Thread-12:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 20:41:32,613 [Thread-12:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.026) SELECT `goods`.`id`, `goods`.`user_id`, `goods`.`shop_id`, `goods`.`name`, `goods`.`pgoods_id`, `goods`.`sendaddress`, `goods`.`platform`, `goods`.`image1`, `goods`.`image2`, `goods`.`image3`, `goods`.`keyword1`, `goods`.`price1`, `goods`.`remark1`, `goods`.`keyword2`, `goods`.`price2`, `goods`.`remark2`, `goods`.`keyword3`, `goods`.`price3`, `goods`.`remark3`, `goods`.`keyword4`, `goods`.`price4`, `goods`.`remark4`, `goods`.`keyword5`, `goods`.`price5`, `goods`.`remark5`, `goods`.`add_time` FROM `goods` WHERE `goods`.`id` = '1501998367719358'; args=('1501998367719358',)
2017-08-06 20:41:32,646 [Thread-12:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.026) SELECT `cryapp_cryorder`.`id`, `cryapp_cryorder`.`GoodId_id`, `cryapp_cryorder`.`ShopId_id`, `cryapp_cryorder`.`Userid_id`, `cryapp_cryorder`.`buyerid_id`, `cryapp_cryorder`.`managerid_id`, `cryapp_cryorder`.`tbUsername_id`, `cryapp_cryorder`.`jdUsername_id`, `cryapp_cryorder`.`Money`, `cryapp_cryorder`.`Express`, `cryapp_cryorder`.`sellerMoney`, `cryapp_cryorder`.`buyerMoney`, `cryapp_cryorder`.`Keywords`, `cryapp_cryorder`.`platform`, `cryapp_cryorder`.`OrderSort`, `cryapp_cryorder`.`PlatformOrdersid`, `cryapp_cryorder`.`Status`, `cryapp_cryorder`.`StartTime`, `cryapp_cryorder`.`EndTime`, `cryapp_cryorder`.`AddTime`, `cryapp_cryorder`.`Note` FROM `cryapp_cryorder` WHERE `cryapp_cryorder`.`GoodId_id` = '1501998367719358' LIMIT 1; args=('1501998367719358',)
2017-08-06 20:41:32,694 [Thread-12:123145435348992] [django.server:131] [basehttp:log_message] [INFO]- "GET /goods/1501998367719358 HTTP/1.1" 200 18963
2017-08-06 21:04:18,035 [Thread-14:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.317) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:04:18,412 [Thread-14:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.370) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:04:18,423 [Thread-14:123145435348992] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:04:24,535 [Thread-15:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.371) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:04:24,918 [Thread-15:123145435348992] [django.db.backends:90] [utils:execute] [DEBUG]- (0.380) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:04:24,926 [Thread-15:123145435348992] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:10:51,724 [Dummy-6:123145342308352] [django.db.backends:90] [utils:execute] [DEBUG]- (0.015) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:10:51,743 [Dummy-6:123145342308352] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 21:10:52,931 [Dummy-6:123145342308352] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SHOW FULL TABLES; args=None
2017-08-06 21:10:52,973 [Dummy-6:123145342308352] [django.db.backends:90] [utils:execute] [DEBUG]- (0.021) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 21:12:38,398 [Thread-7:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.409) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:12:38,768 [Thread-7:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.369) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:12:38,789 [Thread-7:123145347563520] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:12:47,079 [Thread-8:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.141) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:12:47,203 [Thread-8:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.122) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:12:47,208 [Thread-8:123145347563520] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:12:53,300 [Thread-9:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.348) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:12:53,663 [Thread-9:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.361) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:12:53,669 [Thread-9:123145347563520] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:19:16,783 [Thread-10:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.348) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:19:17,140 [Thread-10:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.353) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:19:17,153 [Thread-10:123145347563520] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:19:20,673 [Thread-11:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.291) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:19:20,795 [Thread-11:123145347563520] [django.db.backends:90] [utils:execute] [DEBUG]- (0.121) SELECT `adapi_advert`.`id`, `adapi_advert`.`name`, `adapi_advert`.`title`, `adapi_advert`.`thumb`, `adapi_advert`.`url`, `adapi_advert`.`timeline` FROM `adapi_advert`; args=()
2017-08-06 21:19:20,801 [Thread-11:123145347563520] [django.server:131] [basehttp:log_message] [INFO]- "GET /adapi/advert HTTP/1.1" 200 865
2017-08-06 21:40:10,331 [Dummy-6:123145373560832] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:40:10,349 [Dummy-6:123145373560832] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 21:40:12,277 [Dummy-6:123145373560832] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 21:40:12,316 [Dummy-6:123145373560832] [django.db.backends:90] [utils:execute] [DEBUG]- (0.020) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 21:40:16,828 [Dummy-6:123145445560320] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:40:16,846 [Dummy-6:123145445560320] [django.db.backends:90] [utils:execute] [DEBUG]- (0.016) SELECT VERSION(); args=None
2017-08-06 21:40:18,864 [Dummy-6:123145445560320] [django.db.backends:90] [utils:execute] [DEBUG]- (0.019) SHOW FULL TABLES; args=None
2017-08-06 21:40:18,888 [Dummy-6:123145445560320] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
2017-08-06 21:43:01,126 [Dummy-6:123145552691200] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT @@SQL_AUTO_IS_NULL; args=None
2017-08-06 21:43:01,145 [Dummy-6:123145552691200] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SELECT VERSION(); args=None
2017-08-06 21:43:01,936 [Dummy-6:123145552691200] [django.db.backends:90] [utils:execute] [DEBUG]- (0.017) SHOW FULL TABLES; args=None
2017-08-06 21:43:01,960 [Dummy-6:123145552691200] [django.db.backends:90] [utils:execute] [DEBUG]- (0.018) SELECT `django_migrations`.`app`, `django_migrations`.`name` FROM `django_migrations`; args=()
